swingset/src/main/java/com/nqadmin/swingset/datasources/RowSetOps.java

```diff
@@ -470,7 +470,7 @@ public static void updateColumnText(final RowSet _rowSet, final String _updatedV
 			
 		case DATE:
 // TODO Good to get rid of getSQLDate if possible.
-			if (_updatedValue.length() == 10) {
+			if (_updatedValue!=null && _updatedValue.length() == 10) {
```

Comment:
The logic, I guess too difficult for the compiler to follow, insures that this is not null (at least its supposed to). I was getting some null warnings, so add the `assert(_updatedValue != null)` to get rid of them. Maybe can take out the assert now.

Reply:
Agree. I didn't scroll up enough to see the assertion. I suppressed the warning and added a TODO.